tests/unit_node/worker_threads_test.ts

```diff
@@ -516,6 +516,29 @@ Deno.test({
   },
 });
 
+Deno.test({
+  name: "[node/worker_threads] Returns terminate promise with exit code",
+  async fn() {
+    const deferred = Promise.withResolvers<void>();
+    const worker = new workerThreads.Worker(
+      `parentPort.postMessage("ok");`,
```

Comment:
I believe you need to import it explicitly

Reply:
Ohh 💩🙈